README.md

```diff
@@ -180,3 +186,18 @@ S3 bucket name, and the credentials the Ingest Service will use to connect to S3
 for the AWS Access Key and the AWS Secret Key. The key values are stored in files called `aws_s3_access.sec` and
 `aws_s3_secret.sec`. These files must be in the same directory as the `docker-compose.yml` and will not be version
 controlled.
+
+Example configuration file:
+```yaml
+aws:
+s3:
+	endpointUrl: https://s3.us-west-1.amazonaws.com
+	region: us-west-1
+	bucket:
+	quarantine: ingest-qurantine
+transformation:
+transformEndpointUrl: http://localhost:1231/transform/
+ingest:
+callbackEndpointUrl: http://localhost.com:1232/store/
+retrieveEndpointUrl: http://localhost.com:1233/retrieve/
```

Comment:
✏️ The indenting here could be improved.

Reply:
I wasn't able to easily fix this. Spotless either removes the spaces or replaces them with tabs, even though yaml doesn't allow tabs. One suggestion is to link to an example yaml file.

Reply:
See https://github.com/connexta/multi-int-store/pull/47/commits/4f8e16025793c2c5c78d12b6272d2821aa0c0734.